### PR TITLE
Related inline field change buttons display & handling (618, 639, 74)

### PR DESCRIPTION
**Pull request by AndreiHondrari**
Django Version: **1.10.7**
Base branch: **stable/2.9.x**

This solves: 
* https://github.com/sehmaschine/django-grappelli/issues/618
* https://github.com/sehmaschine/django-grappelli/issues/639
* https://github.com/theatlantic/django-nested-admin/issues/74

### CODE REVIEW

**Comment by AndreiHondrari** [comment]
Not sure if more changes are required on : https://github.com/sehmaschine/django-grappelli/pull/819/files#diff-c26c185ae4abda98847e6f5404c4c474 @sehmaschine what do you think ?

File: grappelli/static/admin/js/admin/RelatedObjectLookups.js [hunk 1]
```diff
@@ -179,10 +182,7 @@
                 updateRelatedObjectLinks(this);
             }
         });
-        // GRAPPELLI CUSTOM
-        /* triggering select means that update_lookup is triggered with
-        generic autocompleted (which would empty the field) */
```
Comment (AndreiHondrari) [inline grappelli/static/admin/js/admin/RelatedObjectLookups.js]: apparently it does not empty it.

**Comment by sehmaschine** [comment]
thx for the pull request. I'll take a look.

**Comment by rossmechanic** [comment]
+1

**Comment by mykljohn** [comment]
+1

**Comment by st4lk** [comment]
Have additional fix to this PR: https://github.com/AndreiHondrari/django-grappelli/pull/1

**Comment by sehmaschine** [comment]
@st4lk can you please send a complete PR. I'm not sure what exactly is your "fix" (besides removing the comments).

File: grappelli/static/admin/js/admin/RelatedObjectLookups.js (hunk 1 above)
Comment (sehmaschine) [reply grappelli/static/admin/js/admin/RelatedObjectLookups.js]: With my setup it does (with generic autocompletes).

**Comment by sehmaschine** [comment]
@AndreiHondrari Your PR seems to work besides line 187 with RelatedObjectLookup.js – it seems that we need to remove the comment there, but this would lead to empty the field ObjectID with generic autocompletes.

**Comment by sehmaschine** [comment]
I've updated stable/2.12.x in order to resolve this issue. Please take a look and let me know if there are any further issues (styling of the buttons still needs to be done though).

**Comment by sehmaschine** [comment]
Did you see that I already committed a solution with stable/2.12.x? Maybe
you want to check that out and see if anything needs to be improved.


Am Sa., 22. Dez. 2018, 23:58 hat Alexey Evseev <notifications@github.com>
geschrieben:

**Comment by sehmaschine** [comment]
Solved with 2.12.2.

**Pull request closed without merging**